x-pack/plugins/file_upload/server/analyze_file.tsx

```diff
@@ -22,6 +22,8 @@ export async function analyzeFile(
   const body = await client.asInternalUser.textStructure.findStructure(
     {
       body: data,
+      // @ts-expect-error TextStructureFindStructureRequest type is out of date and doesn't include ecs_compatibility
```

Comment:
Raised https://github.com/elastic/elasticsearch-specification/issues/1830 to update the request type